google/data_source_stackdriver_ip_ranges.go

```diff
@@ -0,0 +1,55 @@
+package google
+
+import (
+	"fmt"
+	"log"
+	"sort"
+	"time"
+
+	"github.com/hashicorp/terraform/helper/schema"
+	monitoring "google.golang.org/api/monitoring/v3"
+)
+
+func dataSourceStackdriverIpRanges() *schema.Resource {
```

Comment:
Since the actual API is called `uptimeCheckIps`, would it make sense to call this that instead of `IpRanges`?